tests/lib.rs

```diff
@@ -0,0 +1,16 @@
+extern crate libftd2xx_ffi;
+
+use libftd2xx_ffi::*;
+
+#[test]
+#[cfg(not(windows))]
+fn version() {
+    let mut dw_library_ver = 0;
+
+    unsafe {
```

Comment:
This returns a status value that should be checked as well.
```suggestion
    let status = unsafe {
```

Then with the other assert
```rust
assert_eq!(status, 0);
```